### PR TITLE
Explorer description list UI component

### PR DESCRIPTION
**Pull request by Jibz1**

### CODE REVIEW

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import type { ReactNode } from 'react';
+
+import { Text } from '~/ui/Text';
+
+export type LabelProps = ExtractProps<typeof Text>;
+
+export function Label(props: LabelProps) {
+    return (
+        <dt className="col-span-1">
+            <Text {...props}>{props.children}</Text>
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply apps/explorer/src/ui/DescriptionList.tsx:15]: Ok.

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        </dt>
+    );
+}
+
+export function Value({ children }: { children: ReactNode }) {
+    return <dd className="ml-0 col-span-2">{children}</dd>;
+}
+
+export type DescriptionListProps = {
+    children: ReactNode[];
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply apps/explorer/src/ui/DescriptionList.tsx]: done

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type ExtractProps } from './types';
+
+import type { ReactNode } from 'react';
+
+import { Text } from '~/ui/Text';
+
+export type LabelProps = ExtractProps<typeof Text>;
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply apps/explorer/src/ui/DescriptionList.tsx]: done

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ReactNode } from 'react';
+
+import { Text, type TextProps } from '~/ui/Text';
+
+export function Label(props: TextProps) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/ui/DescriptionList.tsx:8]: You should still create and export `LabelProps`, it should just be `export interface LabelProps extends TextProps {}`

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <dt className="col-span-1">
+            <Text variant="body" {...props}>
+                {props.children}
+            </Text>
+        </dt>
+    );
+}
+
+export function Value({ children }: { children: ReactNode }) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/ui/DescriptionList.tsx:18]: Should create and export `ValueProps`

File: apps/explorer/src/ui/DescriptionList.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+import type { ReactNode } from 'react';
+
+import { Text, type TextProps } from '~/ui/Text';
+
+export function Label(props: TextProps) {
+    return (
+        <dt className="col-span-1">
+            <Text variant="body" {...props}>
+                {props.children}
```
Comment (Jordan-Mysten) [inline apps/explorer/src/ui/DescriptionList.tsx:12]: props already has children, so you don't need this separate children carve out: 
```tsx
<Text variant="body" {...props} />
```

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**